backend/src/apiserver/storage/experiment_store.go

```diff
@@ -100,6 +136,16 @@ func (s *ExperimentStore) scanRows(rows *sql.Rows) ([]*model.Experiment, error)
 	return experiments, nil
 }
 
+func (s *ExperimentStore) scanRowToCount(rows *sql.Rows) (int32, error) {
```

Comment:
Is this used to count the total number of rows in the table for all pages? Or just for one page?

Reply:
It counts the total number of rows matching the query, with no regard to pagination.